Havit.Blazor.Components.Web.Bootstrap/Havit.Blazor.Components.Web.Bootstrap.csproj

```diff
@@ -38,6 +38,7 @@
 	
 	<ItemGroup>
 		<PackageReference Include="Microsoft.Extensions.Localization" Version="7.0.10" />
+		<PackageReference Include="Microsoft.Bcl.TimeProvider" Version="8.0.0-rc.1.23419.4" />
```

Comment:
We don't need the reference here, if it is referenced from `Havit.Blazor.Components.Web`.